### DISCUSSION TITLE
Cookie Usage 🍪

### DISCUSSION THREAD

**Discussion by superstes** [piccolo-orm/piccolo_theme#85] [Q&A]
Greetings.

Thanks again for the great theme 😁 

I found that it sets some cookies: https://radar.cloudflare.com/scan/def566ef-c608-4b7b-83c1-5578342e484b/cookies

Is there any information of what they are used for? GDPR likes to know (;

Thanks

**Comment by dantownsend** [comment]
I'm not sure to be honest.

I thought maybe we were pulling in a JS library from a CDN, but looking at the code I can't see anything.

When I look at my Piccolo theme when hosted on Read the Docs, they seem to inject some other stuff.

Are you hosting it yourself, or on something like Read the Docs?

**Reply by superstes** [reply]
I'm hosting it on my own infra.
Just using `sphinx build -b html ...` and serving the static files via nginx
Reference: https://github.com/o-x-l/docs

**Reply by dantownsend** [reply]
Ok, interesting. I'll try again later on with `sphinx build`.

**Reply by superstes** [reply]
Thank you. 🙏
Would be interesting what that is.
Maybe related to the roboto fonts (?)

**Reply by superstes** [reply]
Oh sorry to waste your time..
Was some leftover nginx config 🫣
